.travis.yml

```diff
@@ -1,31 +1,37 @@
+---
 language: python
 python:
-  - "3.6"
-  - "3.6-dev"
-  - "3.7-dev"
-  - "nightly"
+    - '3.6' # EOL 2021-12-23
+    - '3.7' # EOL 2023-06-27
+    - '3.8' # EOL 2024-10
+    - 3.9-dev # RC1 2020-09-10
```

Comment:
Should this 3.9 and add 3.10-dev?

Reply:
That sounds like a follow up at some point.  This should have landed ages ago.